doc/man/8/cephfs-top.rst

```diff
@@ -36,6 +36,14 @@ Options
 
    Perform a selftest. This mode performs a sanity check of ``stats`` module.
 
+.. option:: --conffile [CONFFILE]
+
+   Path to cluster configuration file
+
+.. option:: -d [DELAY], --delay [DELAY]
+
+   Metric refresh interval in seconds (default: 1 second)
```

Comment:
`Metrics refresh interval` instead?

Reply:
Actually I think it would be better as just "Refresh interval in seconds (default: 1)"